src/main/steps/c100-rebuild/check-your-answers/helpers/miam-root-helpers/domesticVoilence.Helper.ts

```diff
@@ -0,0 +1,26 @@
+/* eslint-disable @typescript-eslint/explicit-module-boundary-types */
+
+export const miamExemptionDomesticVoilenceMapperForKeys = (userCase, keys, userKey) => {
+  if (userCase.hasOwnProperty(userKey)) {
+    return userCase[userKey]
+      .filter(field => field !== '')
+      .map(item => {
+        return '<li class="govuk-!-padding-top-1 govuk-!-padding-bottom-1">' + keys[item] + '</li>';
+      });
+  }
+};
+
+export const miamExemptionDomesticVoilenceMapper = (userCase, keys) => {
+  if (userCase.hasOwnProperty('miam_domesticAbuse')) {
```

Comment:
can we make this method as generic and pass the miam_domesticAbuse as parameter. so that it can be used for all fields.